data-raw/DATASET.R

```diff
@@ -67,7 +78,9 @@ df_patient_characteristics <-
   labelled::set_variable_labels(
     patient_id = "Patient ID",
     trt = "Treatment Group",
-    age = "Patient Age"
+    age = "Patient Age",
+    marker = "Marker",
```

Comment:
```suggestion
    marker = "Biological Marker",
```

I wonder if `Marker` by itself could use a bit more context? I don't have strong opinions here.